en/docs/reference/connectors/facebook-ads-connector/facebook-ads-configuration.md

```diff
@@ -0,0 +1,727 @@
+# Facebook Ads Connector Reference
+
+The following operations allow you to work with the Facebook Ads API. Click an operation name to see parameter details and samples on how to use it.
+
+## Initialize the connector
+
+To use the Facebook Ads connector, first create the connection with your configuration. When calling a Facebook Ads operation, ensure the connection is referenced using the `configKey` attribute.
+
+??? note "init"
+    The `init` operation initializes the connection to Facebook Ads API.
```

Comment:
Shall we follow the same format to introduce other operations as well?

Reply:
Fixed with [29f155c](https://github.com/wso2/docs-mi/pull/907/commits/29f155c597b09440cfc9288e66376371c089c06e)